src/components/article/lectorreview/reviewheading/ReviewHeading.js

```diff
@@ -63,6 +47,27 @@ export function ReviewHeading({ propAndChildrenInput }) {
     </Section>
   );
 }
+
+export function ReviewHeadingLink({ propAndChildrenInput, className }) {
+  const workTypeTranslated = workTypeTranslator(
+    propAndChildrenInput?.workTypes
+  );
+  return (
+    <div className={`${styles.back_button}  ${className}`}>
+      <IconLink
+        href={templateForHeaderWorkCard(propAndChildrenInput).link_href}
+        border={{ bottom: true, top: false }}
+        iconSrc={ChevronLeft}
+      >
+        {[
+          Translate({ context: "general", label: "back-to" }),
```

Comment:
Lille detalje men synes det er lidt for fancy for the job. 

Istedet for at lave et array for at joine det bagefter, kunne du bare skrive:

`{Translate(...)} {workTypeTranslated}`

Reply:
tjek - det er noget kode akri har skrevet - jeg har bare trukket den del ud jeg skulle bruge